.github/workflows/tests.yaml

```diff
@@ -9,9 +9,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.10", "3.11"]
     env:
-      EIA_API_KEY: ${{ secrets.EIA_API_KEY }}
+      EIA_API_KEY: ${{ vars.EIA_API_KEY }}
```

Comment:
@kmax12 Can you/have you added the `EIA_API_KEY` as a variable? I'm unable to view the repo settings.

Reply:
It should be under Settings > Secrets and Variables > Actions then the Variables tab.

<img width="1095" alt="Screenshot 2024-01-21 at 14 00 45" src="https://github.com/kmax12/gridstatus/assets/17484616/5f838600-737c-4329-82b0-4baf8239b369">